sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Samples/Sample1_AnalyzeConversation.cs

```diff
@@ -33,13 +33,10 @@ public void AnalyzeConversation()
 
             Console.WriteLine($"Top intent: {response.Value.Prediction.TopIntent}");
             #endregion
-
-            Assert.That(response.GetRawResponse().Status, Is.EqualTo(200));
```

Comment:
Why remove these? You should assert something - ideally, something from the expected response itself, though I can understand that can be difficult in cases like this.

Reply:
I think we should be asserting in tests instead of samples, that's why I removed it. I've also made some tests with asserting the answer but haven't pushed them yet. Thought it would be better if I made a separate PR after merging this one.

Reply:
Samples are tests, and designed as such. We want to make sure the test compile and actually work. Only the code between the regions will be extract. Please add the asserts back in.